src/memory.cpp

```diff
@@ -291,6 +296,20 @@ void emu_write(uint8_t reg, uint8_t value)
 		case 5: gif_recorder_set((gif_recorder_command_t)value); break;
 		case 6: wav_recorder_set((wav_recorder_command_t)value); break;
 		case 7: Options.no_keybinds = v; break;
+		case 8: clock_base = clockticks6502; break;
```

Comment:
Windows build needs an explicit cast of `clockticks6502` to `uint32_t`

Reply:
chose to make the snapshot variables  64 bits too